README.md

```diff
@@ -36,14 +36,23 @@ return [
     MonsieurBiz\SyliusSalesReportsPlugin\MonsieurBizSyliusSalesReportsPlugin::class => ['all' => true],
 ];
 ```
+<!-- Since SyliusSalesReportsPlugin does not contain a flex methode you have to manually create this file :  -->
 
-Finally import the routes in `config/routes/monsieurbiz_sylius_sales_reports_plugin.yaml` : 
+Create a file in `sylius/config/packages/sylius_sales_reports.yaml` within :
```

Comment:
`Create a file in `sylius/config/packages/sylius_sales_reports.yaml` within:`

Reply:
I disagree with this change. The path is completely wrong. It should be in `routes/` and the filename is correct as `monsieurbiz_sylius_sales_reports_plugin.yaml`.

Also, the sentence is correct as well.